rebar.config.erl

```diff
@@ -128,30 +128,34 @@ prod_compile_opts() ->
 prod_overrides() ->
     [{add, [ {erl_opts, [deterministic]}]}].
 
-relup_deps(Profile) ->
-    {post_hooks, [{"(linux|darwin|solaris|freebsd|netbsd|openbsd)", compile, "scripts/inject-deps.escript " ++ atom_to_list(Profile)}]}.
+relup_deps(Profile, Vsn) ->
+    InjectCmd = "scripts/inject-relup.escript " ++ filename:join(["_build", Profile, "rel", "emqx"]) ++ " " ++ Vsn,
+    {post_hooks,
+        [ {"(linux|darwin|solaris|freebsd|netbsd|openbsd)", relup, InjectCmd}
+        ]
+    }.
```

Comment:
this does not have to be a rebar hook.
add it to the `make_zip` function in the `build` script.
reason:
the relup file is already built and there is only one release to deal with.
i.e. so there is no need to do the wildcard below
`RelFiles = filelib:wildcard(filename:join([RelRootDir, "releases", "*", "emqx.rel"])),`

Reply:
As I explained below, the collection for emqx vsn is necessary. So I don't know if we still need to put this script to `make_zip`.